### PR TITLE
Added multiple tables

### PR DESCRIPTION
**Pull request by Dmambo**
## FEATURES ADDED

**HERE ARE WHAT WAS IMPLEMENTED IN THE DATABASE**

- Created a table named owners with the following columns:

        

        - [ ✅] id: integer   
        - [ ✅]  full_name: string
        - [ ✅ ]  age: integer

- Created a table named species with the following columns:

    

        - [ ✅] id: integer 
        - [ ✅] name: string

- Modified animals table

- Inserted the following data into the owners table:

        - [✅ ]   Sam Smith 34 years old.
        - [ ✅]   Jennifer Orwell 19 years old.
        - [ ✅]   Bob 45 years old.
        - [✅ ]   Melody Pond 77 years old.
        - [ ✅]   Dean Winchester 14 years old.
        - [✅ ]   Jodie Whittaker 38 years old.

- Inserted the following data into the species table:

 

        - [✅ ]  Pokemon
        - [ ✅]   Digimon

- Modified the inserted animals so it includes the species_id value:

  

        - [✅ ] If the name ends in "mon" it will be Digimon
        - [✅ ]   All other animals are Pokemon

- Modified the inserted animals to include owner information (owner_id):

        - [ ✅]   Sam Smith owns Agumon.
        - [✅ ]   Jennifer Orwell owns Gabumon and Pikachu.
        - [✅ ]   Bob owns Devimon and Plantmon.
        - [✅ ]   Melody Pond owns Charmander, Squirtle, and Blossom.
        - [ ✅]   Dean Winchester owns Angemon and Boarmon.

- Wrote queries (using JOIN) to answer the following questions:

        - [✅ ]   What animals belong to Melody Pond?
        - [✅ ]   List of all animals that are pokemon (their type is Pokemon).
        - [✅ ]   List all owners and their animals, remember to include those that don't own any animal.
        - [✅ ]   How many animals are there per species?
        - [ ✅]   List all Digimon owned by Jennifer Orwell.
        - [ ✅]   List all animals owned by Dean Winchester that haven't tried to escape.
        - [✅ ]   Who owns the most animals?

**BELOW ARE THE SCREENSHOTS OF THE ABOVE FEATURES ADDED** 📷 

ownerTable
speciesTable
speciestoanimal

removespecies
add ids
use
reference
r-images.githubusercontent.com/110314309/221821961-bfd9cc8e-ba80-43ac-ad5c-e237435cf370.JPG)
insertspecies

modify insa
updateanimal
melody
pokemon
ownersandanimals
count
Jennifer
Dean
highest

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @Dmambo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the README file does not include some of the headers like the `future features` section
Screenshot from 2023-02-28 11-13-31


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Ibrahim, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has incorrect gitflow/ GitHub flow. Kindly note that the merge request for this activity should flow **from feature into development**.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,43 +1,4 @@
...
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
-## Getting Started
-
-This repository includes files with plain SQL that can be used to recreate a database:
-
-- Use [schema.sql](./schema.sql) to create all tables.
-- Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
```
Comment (afanimartin) [inline README.md:13]: - Please add explicit instructions for getting started. Add commands for cloning and running the app locally.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Dmambo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🎉 🎉 🎉 
- Files for for data schema, queries and data entry well implemented as required. Good job
- The right readme template used as recommended

## Required Changes ♻️
_Check the comments under the review._
- Please add screenshots for results of running the queries. You currently have screenshots for queries but not rssult of the queries. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
Hi @Dmambo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**